helm-util/package.md

```diff
@@ -0,0 +1,32 @@
+Package `helm-util` allows the user to consume Helm Charts as plain Jsonnet
+resources. This package implements [Helm support](https://tanka.dev/helm) for
+Grafana Tanka.
+
+### Usage
+
+> **Warning:** [Functionality required](#internals) by this library is still experimental and may break.
+
+The [`helm.template`](#fn-helmtemplate) function to converts a Helm Chart into Jsonnet objects,
+to be consumed by tools like [Grafana Tanka](https://tanka.dev).
+
+Helm Charts are required to be available on the local file system and are
+resolved relative to the file that calls `helm.template`:
+
+```jsonnet
+%s
```

Comment:
This markdown file is a template, maybe rename it to `package.md.tpl` or something so that it is clear? Maybe also make clear that the `README.md` file is rendered, I'm used to (like many) that README is written manually.

Reply:
Will do!